app/src/components/page/Sidebar/index.tsx

```diff
@@ -72,7 +87,189 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
   const { roles } = extraData;
   let location = useLocation();
   let loc = location.pathname.split('/');
-  loc.shift();
+  const [openKeys, setOpenKeys] = React.useState<React.ReactText[]>([]);
```

Comment:
do we still need the location.pathname.split part?

Reply:
its needed [here](https://github.com/OpenSRP/web/pull/395/files#diff-552ba8f96c6d579aa4f53424941f377eeb60e0f25fca23ed7ce15a10ffb330dcR271) when refreshing the page so `openKeys` wont be set. `openKeys` is set when clicking a menu item.